packages/e2e-tests/jest.playwright.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+	preset: 'jest-playwright-preset',
+	testMatch: [ '**/specs/editor/blocks/**/*.js' ],
```

Comment:
Why are we only running blocks tests? Is this only an experimental PR?

Reply:
Yes, this is an experimental PR where just blocks' tests are refactored for running with Playwright. If we decide to continue with Playwright though those refactors can be cherry-picked into proper PRs.

Reply:
Ah right, I wanted to keep it as a draft but didn't know why the CI E2Es wasn't running so I opened the PR. Turned out that there was one conflict that was preventing it from running (TIL). I'll switch it back to draft 👍